atlas_scripts/kim_mouse_isotropic.py

```diff
@@ -0,0 +1,262 @@
+ATLAS_LINK = "https://figshare.com/ndownloader/articles/25750983/versions/1"
+
+# The orientation of the **original** atlas data, in BrainGlobe convention:
+# https://brainglobe.info/documentation/setting-up/image-definition.html#orientation
+ORIENTATION = "rsp"
+
+# The id of the highest level of the atlas. This is commonly called root or
+# brain. Include some information on what to do if your atlas is not
+# hierarchical
+ROOT_ID = 997
+
+# The resolution of your volume in microns. Details on how to format this
+# parameter for non isotropic datasets or datasets with multiple resolutions.
+RESOLUTION = 20
+
+ATLAS_PACKAGER = "Pavel Vychyk / pavel.vychyk@brain.mpg.de"
+
+# Custom globals for the retrieved file names
+ONTOLOGY_FILE = "UnifiedAtlas_Label_ontology_v2.csv"
+ANNOTATION_FILE = "UnifiedAtlas_Label_v2_20um-isotropic.nii"
+REFERENCE_FILE = "UnifiedAtlas_template_coronal_20um-isotropic.nii"
+
+
+def download_resources() -> Tuple[Path, Path]:
+    """Download the atlas resource files and create required directories.
+
+    Returns
+    -------
+    Tuple[Path, Path]
+        The root directory and the nested download directory as Path objects.
+    """
+    bg_root_dir = Path.home() / "brainglobe_workingdir" / ATLAS_NAME
+    bg_root_dir.mkdir(parents=True, exist_ok=True)
+
+    download_dir = bg_root_dir / "download"
+    download_dir.mkdir(exist_ok=True)
+    download_dir_content = {item.name for item in download_dir.iterdir()}
+    if (
+        ONTOLOGY_FILE not in download_dir_content
+        and REFERENCE_FILE not in download_dir_content
+        and ANNOTATION_FILE not in download_dir_content
+    ):
+        pooch.retrieve(
+            url=ATLAS_LINK,
+            path=(download_dir),
+            progressbar=True,
+            known_hash=None,
+            processor=pooch.Unzip(extract_dir=download_dir),
+        )
+    return bg_root_dir, download_dir
+
+
+def retrieve_reference_and_annotation(
+    download_dir: Path,
+) -> Tuple[np.ndarray, np.ndarray]:
+    """Retrieve the reference and annotation volumes as two numpy arrays.
+
+    Returns
+    -------
+    Tuple[np.ndarray, np.ndarray]
+        The reference volume and the annotation volume as numpy arrays.
+    """
+    download_dir_content = {item.name for item in download_dir.iterdir()}
+    if ANNOTATION_FILE not in download_dir_content:
+        raise FileNotFoundError(
+            f"{ANNOTATION_FILE} not found in {download_dir}"
+        )
+    if REFERENCE_FILE not in download_dir_content:
+        raise FileNotFoundError(
+            f"{REFERENCE_FILE} not found in {download_dir}"
+        )
+    # Load reference volume (nifti -> numpy array)
+    nifti_img = nib.load(download_dir / REFERENCE_FILE)
+    reference = np.asarray(nifti_img.dataobj).squeeze().astype(np.int32)
+    # Load annotation volume (nifti -> numpy array)
+    nifti_img = nib.load(download_dir / ANNOTATION_FILE)
+    annotation = np.asarray(nifti_img.dataobj).squeeze().astype(np.int32)
```

Comment:
With apologies for nitpicking: could we, for consistency, use `brainglobe_utils.IO.image.load_any` here:
instead of  `nib.load`.

Reply:
(I know it [uses `nibabel` under the hood](https://github.com/brainglobe/brainglobe-utils/blob/24900dcff510dc4450231e93f7cbbc03659b4d12/brainglobe_utils/IO/image/load.py#L225))